src/removeStopwords.js

```diff
@@ -0,0 +1,38 @@
+// SUGGESTIONS: refacture ino shouldDisplay
+// and this function take a word and the language, then it returns true or false, and this function should be used wherever you are trying to display the words
+
+import {  STOPWORDS } from './constants';
```

Comment:
better to save STOPWORDS in a separate file as it is localization not code constants

Reply:
jep